service_directory/tests/test_admin.py

```diff
@@ -131,6 +131,13 @@ def test_location_coords_field_validation(self):
 
         self.assertContains(response, 'Invalid coordinates')
 
+    def test_location_coords_field_validation_when_blank(self):
+        data = self.get_post_payload_for_test_organisation()
+        data['location_coords'] = ''
+
+        response = self.client.post(self.api_url, data)
```

Comment:
I think include a check on the status code here, as `assertNotContains` is quite dangerous as a stand alone test, e.g. if it returns another kind of error message